scripts/test_translation_pipeline.py

```
@@ -0,0 +1,489 @@
+        manager1.register_agent_capability("persist-agent", "en", "es")
+        
+        # Create and start a job
+        source_segments = ["Test persistence"]
+        job_id = manager1.create_translation_job(
+            document_id="persist-doc-1",
+            source_language="en",
+            target_language="es",
+            source_segments=source_segments
+        )
+        
+        manager1.start_translation_job(job_id, ["persist-agent"])
```

Pylint (4.0.2)

[error] No value for argument 'target_languages' in method call

(E1120)